man/all_linters.Rd

```diff
@@ -0,0 +1,35 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/with.R
+\name{all_linters}
+\alias{all_linters}
+\title{Create a linter configuration based on all available linters}
+\usage{
+all_linters(packages = "lintr", ...)
+}
+\arguments{
+\item{packages}{A character vector of packages to search for linters.}
+
+\item{...}{Arguments of elements to change. If unnamed, the argument is automatically named.
+If the named argument already exists in the list of linters, it is replaced by the new element.
+If it does not exist, it is added. If the value is \code{NULL}, the linter is removed.}
+}
+\description{
+Create a linter configuration based on all available linters
+}
+\examples{
+names(all_linters())
+
+}
+\seealso{
+\itemize{
+\item \link{linters_with_defaults} for basing off lintr's set of default linters.
+\item \link{linters_with_tags} for basing off tags attached to linters, possibly across multiple packages.
+\item \link{available_linters} to get a data frame of available linters.
+\item \link{linters} for a complete list of linters available in lintr.
+}
+
+Other linter-config: 
+\code{\link{linters_with_tags}()},
+\code{\link{modify_defaults}()}
```

Comment:
This looks quite odd (duplicated links)

Reply:
Removed the family tag. Redundant when the seealso tag is also used.